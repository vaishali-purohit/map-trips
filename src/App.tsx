import './App.css';

import GMap from './containers/GMap';
import Header from './containers/Header';
import React from 'react';

// eslint-disable-next-line @typescript-eslint/no-explicit-any
const App = () => {
  return (
    <div className='App'>
      <Header />
      <GMap />
    </div>
  );
}

export default App;
