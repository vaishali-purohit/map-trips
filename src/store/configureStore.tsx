import configureStoreDev from './configureStore.dev'
import configureStoreProd from './configureStore.prod'

// eslint-disable-next-line @typescript-eslint/no-explicit-any
const selectedConfigureStore: any =
  process.env.NODE_ENV === 'development'
    ? configureStoreDev
    : configureStoreProd

export const { configureStore } = selectedConfigureStore

export const { history } = selectedConfigureStore
