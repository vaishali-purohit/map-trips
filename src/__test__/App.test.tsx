import 'jest-styled-components';

import App from '../App';
import { Provider } from 'react-redux';
import React from 'react';
import configureStore from 'redux-mock-store';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('With React Testing Library', () => {
  const initialState = { output: 10 };
  const mockStore = configureStore();
  let store;

  it('should render correctly in "debug" mode', () => {
    store = mockStore(initialState);
    const component = shallow(
      <Provider store={store}>
        <App />
      </Provider>,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
