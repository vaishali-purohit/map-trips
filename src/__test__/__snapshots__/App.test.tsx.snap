// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With React Testing Library should render correctly in "debug" mode 1`] = `
<ContextProvider
  value={
    {
      "getServerState": undefined,
      "store": {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": {
        "addNestedSub": [Function],
        "getListeners": [Function],
        "handleChangeWrapper": [Function],
        "isSubscribed": [Function],
        "notifyNestedSubs": [Function],
        "trySubscribe": [Function],
        "tryUnsubscribe": [Function],
      },
    }
  }
>
  <App />
</ContextProvider>
`;
